xExtension-YouTube/i18n/en/ext.php

```diff
@@ -3,9 +3,14 @@
 return array(
 	'yt_videos' => array(
 		'height' => 'Player height',
-        'width' => 'Player width',
-        'updates' => 'You can find the latest extension version at',
-		'show_content' => 'Display the feeds content',
-        'use_nocookie' => 'Use the cookie-free domain www.youtube-nocookie.com',
+		'width' => 'Player width',
```

Comment:
I know, this one is annoying ;-) but it has to be changed. 
And it is not up to discussion, because every developer has to decide that one for himself ... all of my projects use 4 spaces for one tab.

Reply:
You want me to use 4 spaces instead of tabs ? It's okay, but just for my own defense (:p) those files were already mixing tabs & spaces and configure.ptml use tabs all around ^^'

Reply:
Wow, you are right. I can't believe I committed tabs. Keep it then as is. If more files have to be changed, its beer to have all format changes in one single commit.